src/main/java/Event.java

```diff
@@ -0,0 +1,13 @@
+public class Event extends Task{
+    protected String from;
+    protected String to;
+    public Event(String description, String from, String to){
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+    @Override
```

Comment:
You can consider adding a newline between line 8 & 9 to avoid making it look too squashed!